bftengine/src/bftengine/IncomingMsgsStorage.hpp

```diff
@@ -33,6 +33,7 @@ class IncomingMsgsStorage {
   virtual bool isRunning() const = 0;
 
   virtual void pushExternalMsg(std::unique_ptr<MessageBase> msg) = 0;
+  virtual void pushExternalMsgRaw(char* msg, size_t& size) = 0;
```

Comment:
Why pass (size_t& size) by reference? It does not change inside pushExternalMsgRaw.